### ISSUE TITLE
Correction in Hindi Phrases

### ISSUE THREAD

**Issue by bazingarj** [ausi/slug-generator#16]
मिठाई - mithai  (coming up as mitha-i)
खुशबू - khushbu ( coming up as khasaba)
लेना - lena ( coming up as lana)
पैसे - paise (comping up as pasa)
अब - aba (must be ab)

**Comment by ausi** [comment]
The transliteration between scripts, like Devanagari to Latin in this case, is performed by the [ICU library](http://userguide.icu-project.org/transforms) which uses the data of the [Unicode CLDR](http://cldr.unicode.org/).

The [Devanagari-Latin](https://github.com/unicode-org/cldr/blob/master/common/transforms/Devanagari-Latin.xml) transform internally transforms to [InterIndic](https://github.com/unicode-org/cldr/blob/master/common/transforms/Devanagari-InterIndic.xml) first and afterwards from [InterIndic to Latin](https://github.com/unicode-org/cldr/blob/master/common/transforms/InterIndic-Latin.xml).

Taking “अब” for example, you can see that “अ” gets transformed to `\uE005` in [Devanagari-InterIndic.xml:20](https://github.com/unicode-org/cldr/blob/13cd95684a52cd18c1238574ca519c335758e503/common/transforms/Devanagari-InterIndic.xml#L20) and “ब” to `\uE02C` in [Devanagari-InterIndic.xml:59](https://github.com/unicode-org/cldr/blob/13cd95684a52cd18c1238574ca519c335758e503/common/transforms/Devanagari-InterIndic.xml#L59).  
The Codepoints `\uE005` and `\uE02C` get assigned to `$wa` in [InterIndic-Latin.xml:21](https://github.com/unicode-org/cldr/blob/13cd95684a52cd18c1238574ca519c335758e503/common/transforms/InterIndic-Latin.xml#L21) and `$ba` in [InterIndic-Latin.xml:60](https://github.com/unicode-org/cldr/blob/13cd95684a52cd18c1238574ca519c335758e503/common/transforms/InterIndic-Latin.xml#L60).  
And finally `$wa` to “a” in [InterIndic-Latin.xml:446](https://github.com/unicode-org/cldr/blob/13cd95684a52cd18c1238574ca519c335758e503/common/transforms/InterIndic-Latin.xml#L446) and `$ba` to “ba” in [InterIndic-Latin.xml:298](https://github.com/unicode-org/cldr/blob/13cd95684a52cd18c1238574ca519c335758e503/common/transforms/InterIndic-Latin.xml#L298).

In short:

```
अ -> \uE005 -> $wa -> a
ब -> \uE02C -> $ba -> ba
```

As I have no knowledge about Devanagari I can’t spot at which point the transformations are wrong.  
It would be great, if you can file a ticket directly at the CLDR: <http://cldr.unicode.org/index/bug-reports>

You can reproduce the issue with a single line of PHP code:

```php
echo \Transliterator::create('Deva-Latn')->transliterate('अब');
```
